bin/ert.in

```diff
@@ -33,41 +21,42 @@ def ensureConfig(func):
 
 
 def runExec(executable , args):
-    exec_args = [ executable ] + args
-    os.execvpe( executable , exec_args , os.environ )
-
+    os.execvp(executable, [executable] + args)
```

Comment:
Note that `execvp` inherits the current environment variables, so it will behave as the previous implementation. `execvpe` is rather used if one wants to have different environment variables than the system.